src/test/java/uk/gov/hmcts/cmc/claimstore/services/document/DocumentManagementServiceTest.java

```diff
@@ -108,16 +118,17 @@ public void shouldDownloadDocumentFromDocumentManagement() {
         URI docUri = URI.create("http://localhost:8085/documents/85d97996-22a5-40d7-882e-3a382c8ae1b4");
 
         when(documentMetadataDownloadClient
-            .getDocumentMetadata(anyString(), anyString(), anyString(), anyString(), anyString())
+            .getDocumentMetadata(anyString(), anyString(), eq(USER_ROLES_JOINED), anyString(), anyString())
```

Comment:
this is unnecessary. The previous code was working fine

Reply:
just to verify user roles being used